menpofit/result.py

```diff
@@ -595,6 +595,8 @@ def __str__(self):
         if self.gt_shape is not None:
             if self.initial_shape is not None:
                 out += "\nInitial error: {:.4f}".format(self.initial_error())
+                if isinstance(self, ParametricIterativeResult) or isinstance(self, MultiScaleParametricIterativeResult):
```

Comment:
I think it would be more general to make this check as `if hasattr(self, 'reconstructed_initial_error')`.